refinery/data_set_manager/search_indexes.py

```diff
@@ -112,50 +112,6 @@ def prepare(self, node):
 
         id_suffix = "_" + id_suffix + "_s"
 
-        # User files gets pre-defined attributes for now.
-        data['filename_Characteristics' + NodeIndex.GENERIC_SUFFIX] = node.name
-
-        data['filetype_Characteristics' + NodeIndex.GENERIC_SUFFIX] = node.type
-
-        data.update(self._assay_data(node))
-
-        # create dynamic fields for each attribute
-        for annotation in annotations:
```

Comment:
shifted the order, did not delete any code. We grab file_store_item first so we can set the user/files index before we create dynamic fields